test/Dockerfile

```diff
@@ -1,10 +1,15 @@
 FROM node:14.21.3-bullseye
 
 RUN apt-get update && \
-    apt-get -y install docker.io chromium python3-pip && \
+    apt-get --no-install-recommends -y install \
+        docker.io=20.10.5+dfsg1-1+deb11u2 \
```

Comment:
The extra version specifications here and in the `pip3` install below are here to quiet warnings. Regardless I do think for this test-runner container it's best if we pin versions anyway.